### PR TITLE
Changelog for 0.28.0

### PR DESCRIPTION
**Pull request by rgrunber**
CC'ing @datho7561 

There's a few issues to fix. https://github.com/eclipse/lemminx/milestone/43?closed=1 (0.27.0) contains some of the issue done for 0.28.0, so we need to create a separate 0.28.0 milestone and move the issues/prs mentioned here into the new one.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Change Log
 
+## [0.28.0](https://github.com/eclipse/lemminx/milestone/44?closed=1) (May 27, 2024)
+
+### Enhancements
+
+ * XML Files support with settings. See [#1464](https://github.com/eclipse/lemminx/issues/1464).
```
Comment (datho7561) [inline CHANGELOG.md:7]: I think we can come up with a more descriptive title for this feature, maybe this:

Setting to specify which attributes should be treated as filepaths for the purpose of editor features

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
 # Change Log
 
+## [0.28.0](https://github.com/eclipse/lemminx/milestone/44?closed=1) (May 27, 2024)
```
Comment (datho7561) [inline CHANGELOG.md:3]: This milestone has not yet been created, and has no closed issues in it. This link is dead

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (rgrunber) [reply CHANGELOG.md:3]: Yup. All the new issues were placed under the previous milestone, which was already released. I don't have commit rights here so can't move them into a new 0.28.0 one but it should be just a matter of taking the issues/PRs in this chanagelog and moving them there.

Comment (datho7561) [reply CHANGELOG.md:3]: Okay, forgot you don't have commit rights here, I can do that, one second

Comment (datho7561) [reply CHANGELOG.md:3]: okay, I think I got it

**Review by datho7561 — APPROVED** [review]

**Pull request merged**